main.js

```diff
@@ -1,35 +1,35 @@
 'use strict';
 
 // THIS IS AN EXAMPLE
```

Comment:
We should delete this comment, its leftover from the electron starter code

Reply:
I'd like to think of our project as a model electron application. but sure i'll remove it